rotkehlchen/exchanges/__init__.py

```diff
@@ -1,6 +1,9 @@
 from rotkehlchen.exchanges.binance import Binance  # noqa: F401
 from rotkehlchen.exchanges.bitmex import Bitmex  # noqa: F401
 from rotkehlchen.exchanges.bittrex import Bittrex  # noqa: F401
+from rotkehlchen.exchanges.coinbase import Coinbase  # noqa: F401
```

Comment:
Thanks. I guess everywhere in the code we use the direct import which is why these were not added there. That was due to bad coding and circular imports. I guess since then refactorings fixed those, so perhaps it's time to do the same in the rest of the codebase!